### PR TITLE
[tests-only] [full-ci] Separated large scripts

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
large script commands used in different steps are separated to `tests/drone`

## Related Issue
- Part of #5231 

## How Has This Been Tested?

- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by phil-davis** [comment]
CI earlier today was OK. But now we are again getting: https://drone.owncloud.com/owncloud/web/16532/58/3
`error An unexpected error occurred: "https://registry.yarnpkg.com/@babel/parser/-/parser-7.14.6.tgz: connect ECONNREFUSED 104.16.26.35:443".`
Trouble with the npm-install step. Something on "the big bad internet" is not happy.

File: .drone.star [hunk 1]
```diff
@@ -1548,10 +1548,7 @@ def buildWebApp():
         "image": "owncloudci/nodejs:14",
         "pull": "always",
         "commands": [
-            "yarn build",
-            "mkdir -p /srv/config",
-            "cp -r %s/tests/drone /srv/config" % dir["web"],
-            "ls -la /srv/config/drone",
+            "bash -x tests/drone/build-web-app.sh {}".format(dir["server"]),
```
Comment (phil-davis) [inline .drone.star:1551]:
```suggestion
            "bash -x tests/drone/build-web-app.sh {}".format(dir["web"]),
```

This used to refer to the "web" dir???

**Review by phil-davis — APPROVED** [review]

File: .drone.star (hunk 1 above)
Comment (phil-davis) [reply .drone.star]: fixed

**Comment by kiranparajuli589** [comment]
hmm strange, some ocis suites failed to request for ocis
```
FetchError: request to https://ocis:9200/ocs/v2.php/cloud/users/Alice?format=json failed, reason: getaddrinfo ENOTFOUND ocis
```

**Comment by phil-davis** [comment]
I restarted CI 15 minutes ago - maybe it will pass this time!?
The last 2 days there have also been problems with npm-install - some issue on "the big bad internet"

**Comment by phil-davis** [comment]
@kiranparajuli589 you can rebase and cleanup #5265 now.

**Pull request merged**